local-cluster/bench/plot.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Plot each transaction confirmation time using gnuplot
+
+set -e
+
+[ $# -eq 1 ] || { echo "Expecting benchmark output directory as argument"; exit 1; }
+
+DIR=$1
+
+cat "$DIR/results.csv" | tr -d \" | tr -s ',' '\t' > ${DIR}/results.data
```

Comment:
So.. tab-separated values?
I'd suggest changing either the haskell code to produce this, or have gnuplot accept the csv.